code/common/datadog.q

```diff
@@ -0,0 +1,36 @@
+// DATADOG CHECKS
+
+//Create datadog namespace
+\d .dg
+
+ddconfigfile:@[value;`ddconfigfile;hsym `$getenv[`KDBAPPCONFIG],"/ddconfig.txt"]
+
+//sets dogstatsd_port to the port defined by ddconfigfile
+$[`ddconfig.txt in key hsym `$getenv[`KDBAPPCONFIG];value each read0 ddconfigfile;dogstatsd_port:8125]
+
+//Functions are set to return 1b or 0b based on the health of the service.
+//Default check returns 1b from each process to indicate process is up and can be queried.
+//These checks run every time crontab/timer is run.
+
+handlers:(`symbol$())!()
+
+is_ok:{[x]
+        f:$[@[{[x].proc.proctype in key .dg.handlers};`;0b];
```

Comment:
This indent is extremely agressive, please tone down to 2 spaces.

I also don't really understand why an error trap is need here, I don't see a scenario where `.proc.proctype` and `.dg.handlers` won't exist at this point. Please correct me if I'm wrong. Can also make this a one liner now that it's slimmed up.
```suggestion
        f:$[.proc.proctype in key handlers;.dg.handlers .proc.proctype;.dg.default_is_ok];
```
I'm also inclined to say that `default_is_ok` doesn't need to be defined and you can use a lambda in its place.

Reply:
Yes the function was more complicated previously which was why there was an error trap. Will change.